manuscript/chapter2-philosophy.md

```diff
@@ -24,6 +24,8 @@ I> Good artists copy; great artists steal.
 I> From: [https://quoteinvestigator.com/2013/03/06/artists-steal/](https://quoteinvestigator.com/2013/03/06/artists-steal/)
 I> In the IT world there is another version: "Reinventing the wheel" but I prefer to add also "What is not understood is not possessed" of Johann Von Goethe to help you understand the philosophy.
 
+The seven stages of Grief is a way to express the various steps of a loss, anyway someone converted as [Seven stages of OpenSource](https://coiled.io/blog/stages-of-openness.html) that can saw as a way to understand how the Open Source process can change your view on doing and understand Open Source (before to read this chapter if you don't want to see my view on this topic).
```

Comment:
The link says „stages of openness” :o

Reply:
I don't get the last part.

*before you read this chapter

but than the condition is confusing.

Perhaps

*Go read that chapter before you want to see my view on this topic

or

*Skip this chapter if you don't want to see (read?) my view on this topic